packages/ethereumjs-config-nyc/package.json

```diff
@@ -15,5 +15,9 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "peerDependencies": {
+    "nyc": "^15.0.0",
```

Comment:
Please use version `< 15` (`^13.2.0` was fine), to produce reports with absolute paths, which is important for our current monorepo setting.

Reply:
I will update, thanks.